cmd/kpod/inspect.go

```diff
@@ -83,7 +83,10 @@ func inspectCmd(c *cli.Context) error {
 	}
 	server, err := libkpod.New(config)
 	if err != nil {
-		return err
+		return errors.Wrapf(err, "could not get container server")
+	}
+	if err = server.Update(); err != nil {
```

Comment:
I can do that.  I'm going to push up a PR with Update() inside of new just so that the tests can run in case the failures are simply because they're running locally